packages/indexer-agent/CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Contract interface changes involving Exponential Rebates
+- Included the protocolNetwork filed when checking for duplicate action targets
+
+### Added
+- Enforced active connections with Graph-Node and the database during startup
```

Comment:
Is this the connection pool fix? If so might be mentioning explicitly that we now allow configuring the db connection pool?